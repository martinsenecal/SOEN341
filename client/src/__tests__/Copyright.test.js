import React from 'react';
import renderer from 'react-test-renderer';
import Copyright from '../components/building-blocks/Copyright';
import {BrowserRouter as Router} from 'react-router-dom';

test('Copyright Test', () => {
  const component = renderer.create(
    <Router>
      <Copyright />
    </Router>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});