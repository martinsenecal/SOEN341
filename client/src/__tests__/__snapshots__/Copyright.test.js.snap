// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright Test 1`] = `
<p
  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
>
  Copyright Â© 
  <a
    color="inherit"
    href="/"
    onClick={[Function]}
  >
    PhotoX
  </a>
   
  2021
  .
</p>
`;
